src/app/pages/categories/pages-transaction-categories.service.ts

```diff
@@ -18,4 +18,20 @@ export class PagesTransactionCategoriesService {
       )),
     );
   }
+
+  public createTransactionCategory(category: TransactionCategory): Observable<TransactionCategory> {
+    return this.domainsTransactionCategoriesGateway.createTransactionCategory(category.toPayload()).pipe(
```

Comment:
Considering that probably this service won't use two gateways you can always use shorter names like `gateway` instead `domainsTransactionCategoriesGateway`. If another gateway will be imported we can always change that.